### PR TITLE
Update description of `--save` to mention saving to keychain access

### PR DESCRIPTION
**Pull request by MuweiHe**
Release Notes
-------------
New Features
- Update description of ``confluent login --save`` to mention that credentials will also be saved to keychain on macOS.

Checklist
---------
1. [CRUCIAL] Is the change for features that are already live in prod?
   * yes: ok

What
----

update description of `--save`

References
----------

https://confluent.slack.com/archives/CG6BW233L/p1677175000331529

Test & Review
-------------

Open Questions / Follow-ups
---------------------------

### CODE REVIEW

File: internal/cmd/login/command.go [hunk 1]
```diff
@@ -68,7 +68,7 @@ func New(cfg *v1.Config, prerunner pcmd.PreRunner, ccloudClientFactory pauth.CCl
 	cmd.Flags().Bool("no-browser", false, "Do not open a browser window when authenticating via Single Sign-On (SSO).")
 	cmd.Flags().String("organization-id", "", "The Confluent Cloud organization to log in to. If empty, log in to the default organization.")
 	cmd.Flags().Bool("prompt", false, "Bypass non-interactive login and prompt for login credentials.")
-	cmd.Flags().Bool("save", false, "Save username/password (non-SSO) credentials to the .netrc file in your $HOME directory. Use the flag to get automatically logged back in when your token expires, after one hour for Confluent Cloud or after six hours for Confluent Platform.")
+	cmd.Flags().Bool("save", false, "Save username/password (non-SSO) credentials to the .netrc file in your $HOME directory, and to Keychain Access on Darwin systems. Use the flag to get automatically logged back in when your token expires, after one hour for Confluent Cloud or after six hours for Confluent Platform.")
```
Comment (brianstrauch) [inline internal/cmd/login/command.go:71]: We're not saving to the .netrc file anymore, right?
```suggestion
	cmd.Flags().Bool("save", false, "Save username and encrypted password credentials (non-SSO credentials) to the configuration file in your $HOME directory, or to Keychain Access on macOS. You will be automatically logged back in when your token expires, after one hour for Confluent Cloud or after six hours for Confluent Platform.")
```

**Review by brianstrauch — CHANGES_REQUESTED** [review]

File: internal/cmd/login/command.go (hunk 1 above)
Comment (MuweiHe) [reply internal/cmd/login/command.go:71]: oh ty for catching! it's actually `and`, on darwin we're using both config and keychain

File: internal/cmd/login/command.go [hunk 2]
```diff
@@ -68,8 +68,7 @@ func New(cfg *v1.Config, prerunner pcmd.PreRunner, ccloudClientFactory pauth.CCl
 	cmd.Flags().Bool("no-browser", false, "Do not open a browser window when authenticating via Single Sign-On (SSO).")
 	cmd.Flags().String("organization-id", "", "The Confluent Cloud organization to log in to. If empty, log in to the default organization.")
 	cmd.Flags().Bool("prompt", false, "Bypass non-interactive login and prompt for login credentials.")
-	cmd.Flags().Bool("save", false, "Save username/password (non-SSO) credentials to the .netrc file in your $HOME directory. Use the flag to get automatically logged back in when your token expires, after one hour for Confluent Cloud or after six hours for Confluent Platform.")
-
+	cmd.Flags().Bool("save", false, "Save username and encrypted password (non-SSO credentials) to the configuration file in your $HOME directory, and to Keychain Access on macOS. You will be automatically logged back in when your token expires, after one hour for Confluent Cloud or after six hours for Confluent Platform.")
```
Comment (brianstrauch) [inline internal/cmd/login/command.go:71]: It's called "Keychain Access" but I think people colloquially know it as "macOS keychain".
```suggestion
	cmd.Flags().Bool("save", false, "Save username and encrypted password (non-SSO credentials) to the configuration file in your $HOME directory, and to macOS keychain if applicable. You will be automatically logged back in when your token expires, after one hour for Confluent Cloud or after six hours for Confluent Platform.")
```

File: go.mod [hunk 3]
```diff
@@ -9,6 +9,7 @@ require (
 	github.com/brianstrauch/cobra-shell v0.4.0
 	github.com/chromedp/chromedp v0.8.7
 	github.com/client9/gospell v0.0.0-20160306015952-90dfc71015df
+	github.com/confluentinc/bincover v0.2.0
```
Comment (brianstrauch) [inline go.mod:12]: This dependency was removed recently so you should be able to delete it!

**Review by brianstrauch — APPROVED** [review]

**Pull request merged**